src/feed/feed.controller.ts

```
@@ -0,0 +1,20 @@
+import { Controller, Post, Param, Request, Get } from '@nestjs/common';
+import { FeedService } from './feed.service';
+ 
+
+@Controller('feed')
+export class FeedController {
+  constructor(private readonly feedService: FeedService) {}
+
+  
+  @Post(':postId/save-toggle')
+  toggleSave(@Param('postId') postId: number, @Request() req) {
+    return this.feedService.toggleSavePost(+postId, req.user.id = 1); // Simulate user.id = 1 if auth not ready
```

ESLint

[error] Replace `req.user.id·=·1` with `(req.user.id·=·1)`

(prettier/prettier)